qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -549,6 +579,263 @@ def __deepcopy__(self, _memo: dict = None) -> "IBMBackend":
+                "'use_measure_esp' is unset or set to 'False'."
+            )
+        actually_dynamic = are_circuits_dynamic(circuits)
+        if dynamic is False and actually_dynamic:
+            warnings.warn(
+                "Parameter 'dynamic' is False, but the circuit contains dynamic constructs."
+            )
+        dynamic = dynamic or actually_dynamic
+
+        if dynamic and "qasm3" not in getattr(self.configuration(), "supported_features", []):
+            warnings.warn(f"The backend {self.name} does not support dynamic circuits.")
+
+        status = self.status()
+        if status.operational is True and status.status_msg != "active":
+            warnings.warn(f"The backend {self.name} is currently paused.")
+
+        program_id = str(run_config.get("program_id", ""))
+        if program_id:
+            run_config.pop("program_id", None)
+        else:
+            program_id = QASM3RUNNERPROGRAMID if dynamic else QOBJRUNNERPROGRAMID
+
+        image: Optional[str] = run_config.get("image", None)  # type: ignore
+        if image is not None:
+            image = str(image)
+
+        if isinstance(init_circuit, bool):
+            warnings.warn(
+                "init_circuit does not accept boolean values. "
+                "A quantum circuit should be passed in instead."
+            )
+
+        if isinstance(shots, float):
+            shots = int(shots)
+
+        run_config_dict = self._get_run_config(
+            program_id=program_id,
+            init_circuit=init_circuit,
+            init_num_resets=init_num_resets,
+            header=header,
+            shots=shots,
+            memory=memory,
+            meas_level=meas_level,
+            meas_return=meas_return,
+            rep_delay=rep_delay,
+            init_qubits=init_qubits,
+            use_measure_esp=use_measure_esp,
+            noise_model=noise_model,
+            seed_simulator=seed_simulator,
+            **run_config,
+        )
+
+        run_config_dict["circuits"] = circuits
+
+        return self._runtime_run(
+            program_id=program_id,
+            inputs=run_config_dict,
+            backend_name=self.name,
+            job_tags=job_tags,
+            image=image,
+        )
+
+    def _runtime_run(
+        self,
+        program_id: str,
+        inputs: Dict,
+        backend_name: str,
+        job_tags: Optional[List[str]] = None,
+        image: Optional[str] = None,
+    ) -> RuntimeJob:
+        """Runs the runtime program and returns the corresponding job object"""
+        hgp_name = None
+        if self._service._channel == "ibm_quantum":
+            hgp_name = self._instance or self._service._get_hgp().name
+
+        session = self._session
+        if session and not session.active:
+            raise RuntimeError(f"The session {session.session_id} is closed.")
+        session_id = session.session_id if session else None
+        start_session = session_id is None
```

Comment:
Wouldn't this also be `True` if I never passed in a Session ?

Reply:
You are right. Added a check for whether there is a session.